pygeo/DVConstraints.py

```diff
@@ -5828,8 +5830,9 @@ def evalCurvArea(self, iSurf):
             # Now compute the KS function for mean curvature, equivelent to KS(H*H*dS)
             sigmaH = np.dot(one, np.exp(self.KSCoeff * H * H * dS))
             KSmean = np.log(sigmaH) / self.KSCoeff
-            if MPI.COMM_WORLD.rank == 0:
-                print("Max curvature: ", max(H * H * dS))
+            # from mpi4py import MPI
```

Comment:
Dont leave stuff commented out like this if the code will be removed. Either include if checks so it doesnt run in certain cases, remove the code alltogether, or leave it in there.